test/e2e/upgrade_clusterctl_test.go

```diff
@@ -11,102 +11,115 @@ import (
 	bmov1alpha1 "github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/cluster-api/cmd/clusterctl/client/config"
 	capi_e2e "sigs.k8s.io/cluster-api/test/e2e"
 	framework "sigs.k8s.io/cluster-api/test/framework"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const workDir = "/opt/metal3-dev-env/"
 
-var _ = Describe(fmt.Sprintf("When testing cluster upgrade from releases %s > current [clusterctl-upgrade]", os.Getenv("CAPM3_FROM_RELEASE")), func() {
+var _ = Describe("When testing cluster upgrade from releases (v1.2=>current) [clusterctl-upgrade]", func() {
 	BeforeEach(func() {
 		osType := strings.ToLower(os.Getenv("OS"))
 		Expect(osType).ToNot(Equal(""))
 		validateGlobals(specName)
-		imageURL, imageChecksum := EnsureImage(e2eConfig.GetVariable("INIT_WITH_KUBERNETES_VERSION"))
+		imageURL, imageChecksum := EnsureImage("v1.23.17")
```

Comment:
why removing the static var here it will be easier to keep this values on e2econfig and just use static var here onthe code

Reply:
now we have multiple clusterctl upgrade tests and each test might use a different k8s version, in this scenario `INIT_WITH_KUBERNETES_VERSION` does not make sense. For example, if you check our `main` branch tests we different versions of k8s for `(v1.6=>current)` and `(v1.5=>current)` :

- `(v1.6=>current)`  https://github.com/metal3-io/cluster-api-provider-metal3/blob/263b0063865a7be5f344a8b288adae07a5f38e16/test/e2e/upgrade_clusterctl_test.go#L28 
- `(v1.5=>current)`  https://github.com/metal3-io/cluster-api-provider-metal3/blob/263b0063865a7be5f344a8b288adae07a5f38e16/test/e2e/upgrade_clusterctl_test.go#L88

Reply:
ok